PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcalEmbeddingHelper.cxx

```diff
@@ -1310,6 +1422,10 @@ Bool_t AliAnalysisTaskEmcalEmbeddingHelper::SetupInputFiles()
       wrapped = true;
     }
 
+    // Accept if the MC file has a run number that belongs 
+    // to the same run block range as the data run used to anchor the MC
+    if ( !IsRunInRunblock(filename->c_str()) ) continue;
```

Comment:
If you remove unwanted filenames, I think this also isn't an issue anymore

Reply:
This is the place where the unwanted files are removed, I have not found the place it is done.
Let's say, I found it simpler to skip the files before adding them to the chain than modifying the vector with the list of files, I was not sure how to do it.